web/packages/core/src/load-ruffle.ts

```diff
@@ -55,8 +61,40 @@ async function fetchRuffle(config: Config): Promise<typeof Ruffle> {
     const { default: init, Ruffle } = await (extensionsSupported
         ? import("../pkg/ruffle_web-wasm_extensions")
         : import("../pkg/ruffle_web"));
+    let response;
+    const wasmUrl = extensionsSupported
+        ? new URL("../pkg/ruffle_web-wasm_extensions_bg.wasm", import.meta.url)
+        : new URL("../pkg/ruffle_web_bg.wasm", import.meta.url);
+    const wasmResponse = await fetch(wasmUrl.href);
+    if (progressCallback) {
+        const contentLength = wasmResponse.headers.get("content-length")!;
+        let bytesLoaded = 0;
+        const bytesTotal = parseInt(contentLength, 10);
+        response = new Response(
+            new ReadableStream({
+                async start(controller) {
+                    const reader = wasmResponse.body?.getReader();
+                    if (!reader) {
+                        throw "Response had no body";
+                    }
+                    progressCallback?.(bytesLoaded, bytesTotal);
+                    for (;;) {
```

Comment:
nit: IMO `while (true)` is more readable and has a more clear meaning.

Reply:
When trying to add `while (true)`, running `npm run format` results in the message:

```
.../ruffle/web/packages/core/src/load-ruffle.ts
  81:28  error  Unexpected constant condition  no-constant-condition

✖ 1 problem (1 error, 0 warnings)
```

Reply:
Should I add to https://github.com/ruffle-rs/ruffle/blob/57659a1f4432d3a4609d23bc2d74349e4e7c3748/web/.eslintrc.yaml:

      no-constant-condition:
        - error
        - checkLoops: false

See https://github.com/eslint/eslint/commit/6ea5d19ec6ccf11076cbc559db818e83e7d1490a

Reply:
Okay, let's keep it as-is.